mwdb/web/src/components/ShowObject/Views/RelationsBox.js

```diff
@@ -3,18 +3,29 @@ import { Link } from "react-router-dom";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 import { APIContext } from "@mwdb-web/commons/api/context";
+import { AuthContext } from "@mwdb-web/commons/auth/context";
 import { ObjectContext } from "@mwdb-web/commons/context";
 import {
     ObjectLink,
     ActionCopyToClipboard,
+    ConfirmationModal,
     TagList,
 } from "@mwdb-web/commons/ui";
 import RelationsAddModal from "../Actions/RelationsAddModal";
 
 function RelationsBox(props) {
     const api = useContext(APIContext);
+    const auth = useContext(AuthContext);
     const context = useContext(ObjectContext);
     const [isAttributeAddModalOpen, setAttributeAddModalOpen] = useState(false);
+    const [isAttributeDeleteModalOpen, setAttributeDeleteModalOpen] = useState(
+        false
+    );
+    const [disabledModalButton, setDisabledModalButton] = useState(false);
+    const [relationToRemove, setRelationToRemove] = useState({
+        relation: "",
+        id: "",
+    });
```

Comment:
Do we need to initialize it with non-empty dict?
```suggestion
    const [relationToRemove, setRelationToRemove] = useState({});
```